### ISSUE TITLE
Gamma vs Exponential pdf

### ISSUE THREAD

**Issue by piggjf** [mavam/stat-cookbook#28]
Gamma distribution does not correspond to the exponential distribution.  It is known that gamma(1,beta) ~ exp(beta), but this isn't the case with these two formulas.  I think the appropriate fix would be to update the gamma distribution to use (1/beta).

**Comment by mavam** [comment]
(Apologies for the late response.)

Are you talking about this identity?

![image](https://user-images.githubusercontent.com/53797/127091035-aa81148b-838e-4347-9d36-bce6e3293808.png)

I'm not quite sure what other reference could be the one you mean. That said, [the sum of n exponential (β) random variables is a gamma (n, β) random variable](https://en.wikipedia.org/wiki/Relationships_among_probability_distributions).

**Comment by piggjf** [comment]
![image](https://user-images.githubusercontent.com/50065875/127091593-5f568402-fb8b-4a49-a09a-b440fa2be1fd.png)

No worries.  It's been a while, but I think I was referring to page 5 of the cookbook where you list the pdf as shown in the screenshot.  I would expect Gamma(1,$\beta$) and Exp($\beta$) to resolve to the same, but this doesn't appear to be the case here.  They seem to differ in their interpretation of $\beta$.  If this was intentional, my apologies for the ticket and please pardon the interruption.  

In any case, this resource has been invaluable as I complete my courses in my MSDS program.  Thank you for taking the time to put it together.

**Comment by mavam** [comment]
I think you talk about the case α = 1 for Gamma(α, β), right? In this case the PDF of the Gamma function evaluates to β/1 * x^0 * e^(-βx), which is exactly the PDF of the Exponential. I do not see the bug quite yet, maybe you could derive it?

(Your screenshot also shows the PDF of the Inverse Gamma, so I'm not exactly sure which PDF you mean. But since this is just the reciprocal of the Gamma, the same should apply.)

**Comment by piggjf** [comment]
That's what I get for trying to reduce the image size.  I grabbed the wrong section.  Here, let me try again with color coding:
![image](https://user-images.githubusercontent.com/50065875/127094054-b63d28d8-3129-403a-bf49-1a65e00fa7a7.png)

For α = 1 in the Gamma (yellow), you get βx^0e^(-βx), but this conflicts with the pdf of the exponential shown on page 5 (green highlighting).

Basically, I think there was a mix up in rate vs scale somehow.

**Comment by mavam** [comment]
Ah, I think I know what you mean now. Basically we don't have a syntax error, but a semantic error: plugging α = 1 results in an Exponential PDF with λ = β, as opposed to λ = 1/β.

Is that what you mean?

**Comment by piggjf** [comment]
Yes, I think that's fair to say.  I just expected a similar interpretation of the parameter across the two distributions.  In case it confused anyone else, I decided to make a ticket to request a modification to the table for the sake of consistency.  It would also make sense if the β were replaced with λ in the exponential pdf.

**Comment by mavam** [comment]
> Yes, I think that's fair to say. I just expected a similar interpretation of the parameter across the two distributions.

Me too.

Changing β to λ in the PDF of the Exponential would have stronger repercussions, unfortunately. So far I went consistently with the notation from statisticians. Changing that would break things in other places.

At this point, I think the best we can do is accept this discrepancy knowingly. 🤷
